CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # Changelog of FOSSology
 
+### 3.8.0 (April 23rd 2020)
+
+This release adds important corrections to
+[3.8.0-rc1](https://github.com/fossology/fossology/releases/tag/3.8.0-rc1)
+
+The release 3.8.0 also introduces new agent `Software Heritage`. There is a
+special note about this agent.
+> Due to rate-limiting from Software Heritage, the agent might run slow. Please
+> check the **Geeky Scan Details** of the agent to understand the cause of the
+> delay.
+
+Please check https://archive.softwareheritage.org/api/#rate-limiting for more
+info.
+
+Some notes about the UTF-8 database. The copyright (and sister) agent now
+creates only UTF-8 string. So it is safe to update to Postgres with UTF-8
+encoded database. For more information, please refer to the wikipage
+[Migration to UTF-8 DB](https://github.com/fossology/fossology/wiki/Migration-to-UTF-8-DB)
+
+#### Credits to contributors for 3.8.0-RC1
+
+From the git commit history, we have following contributors since
+[3.8.0-rc1](https://github.com/fossology/fossology/releases/tag/3.8.0-rc1):
+```
+> adityabisoi <adityabisoi1999@gmail.com>
+> Anupam <ag.4ums@gmail.com>
+> Carmen Bianca Bakker <carmenbianca.bakker@liferay.com>
+> Gaurav Mishra <mishra.gaurav@siemens.com>
+> Kaushl2208 <kaushlendrapratap.9837@gmail.com>
+> Mikko Murto <mikko.murto@gmail.com>
+> sahil <sjha200000@gmail.com>
+> Shaheem Azmal M MD <shaheem.azmal@siemens.com>
+> sjha2048 <sjha200000@gmail.com>
```

Comment:
same mail-id (sjha200000@gmail.com ) two different name can we take one.... or need to take both

Reply:
Removed the `sahil` entry as the correct username is @sjha2048